pkg/wallet/api.go

```diff
@@ -86,6 +91,20 @@ type Pack interface {
 	UnpackMessage(encMessage []byte) (*Envelope, error)
 }
 
+// DIDCreator provide method to create DID document
+type DIDCreator interface {
+	// Creates new DID document.
+	//
+	// TODO :Should be able to specify the DID method as argument (Issue #268).
```

Comment:
#268 is a merged pull request

Reply:
Upcoming PR for issue #268, still in progress.